profile/isa_ro_crate_mapping.md

```diff
@@ -0,0 +1,157 @@
+| ISA term | schema term | Explanation |
+|----------|-------------|-------------|
+|Investigation|Dataset||
```

Comment:
Any way, to distinguish between types and properties? Are "`" being formatted in tables?

Reply:
Or maybe use links to the isa json types and the schema.org types? That would give a direct visual differentiation in addition to the single empty line.

E.g.

|----------|-------------|-------------|
|[Investigation](https://isa-specs.readthedocs.io/en/latest/isajson.html#investigation-schema-json)|[Dataset](https://schema.org/Dataset)||